internal/procedures/server/start.go

```diff
@@ -29,8 +29,11 @@ func Start(initiator common.ServerInitiator) (err error) {
 		return fmt.Errorf("Failed to open the database: '%w'.", err)
 	}
 
+	// Ensure the database closes before a CMD that started a server finishes its execution.
```

Comment:
I wouldn't comment this, since there are multiple places this pattern will be used.